frontend/src/components/P5Wrapper.jsx

```diff
@@ -25,16 +26,51 @@ const P5Wrapper = () => {
       default:
         sketch = Circlefall;
     }
-    const p5Instance = new p5((p) => sketch(p, gamemodeDataFilePath), sketchRef.current);
+    const p5Instance = new p5((p) => sketch(p, gamemodeDataFilePath, setTimer, dispatch), sketchRef.current);
```

Comment:
not sure why we need to pass in a setTimer when we can just use dispatch/context stuff?

Reply:
You're right. My thought process was since its only used once, that it could be easier to understand by having it as `setTimer` rather than a whole dispatch call. But now thinking about it, it doesn't really make sense.